fetcher.py

```diff
@@ -0,0 +1,99 @@
+import os
+import pandas as pd
+
+URL_BASE = (
+    "https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/"
+    "csse_covid_19_data/csse_covid_19_time_series"
+)
+FILENAME_JOHN_HOPKINS = {
+    "confirmed": "time_series_19-covid-Confirmed.csv",
+    "death": "time_series_19-covid-Deaths.csv",
+    "recovered": "time_series_19-covid-Recovered.csv",
+}
+
+URL_COUNTRY_ISO = (
+    "https://gist.githubusercontent.com/tadast/8827699/raw/"
+    "7255fdfbf292c592b75cf5f7a19c16ea59735f74/"
+    "countries_codes_and_coordinates.csv"
+)
+
+MAP_UNMATCHED_COUNTRIES = {
+    'Congo (Kinshasa)': "Congo, the Democratic Republic of the",
+    "Cote d'Ivoire": "Côte d'Ivoire",
+    'Czechia': "Czech Republic",
+    'Eswatini': "Swaziland",
+    'Holy See': "Italy",
+    'Iran': "Iran, Islamic Republic of",
+    'Korea, South': "South Korea",
+    'Moldova': "Moldova, Republic of",
+    'North Macedonia': "Macedonia, the former Yugoslav Republic of",
+    'Reunion': "Réunion",
+    'Taiwan*': "Taiwan",
```

Comment:
Is the star a regexp? It seems like a good idea to use matching. I just don't recognize the syntax.

Reply:
I assume this because John Hopkins mutualize both Tawain area